x-pack/plugins/aiops/common/api/log_categorization/create_category_request.ts

```diff
@@ -64,3 +68,61 @@ export function createCategoryRequest(
     },
   };
 }
+
+const categorizationAnalyzer: AggregationsCustomCategorizeTextAnalyzer = {
+  char_filter: ['first_line_with_letters'],
+  tokenizer: 'standard',
```

Comment:
It would be good to add a comment here saying:

1. This is basically the default categorization analyzer but with the `standard` tokenizer instead of `ml_standard`.
2. The `ml_standard` tokenizer splits tokens in a way that was observed to give better categories in testing many years ago, however, the downside of these better categories is then potential failures to find the original documents when using the category tokens to search for them.
3. Ideally we'd use the tokenizer from the mappings of the field being categorized, but that's too hard, so using `standard` is a quick compromise.

Reply:
Comment added in 287e8688a94ebbc04a34e721e3f58c21ee8af93b